fuel/src/main/kotlin/com/github/kittinunf/fuel/core/RequestExecutionOptions.kt

```diff
@@ -29,6 +31,9 @@ data class RequestExecutionOptions(
     var allowRedirects: Boolean? = null
     var useHttpCache: Boolean? = null
     var interruptCallbacks: MutableCollection<InterruptCallback> = mutableListOf()
+    var responseValidator: ResponseValidator = { response ->
+        !(response.isServerError || response.isClientError)
```

Comment:
This should allow 3xx as a Success in the `Result` value.

Reply:
Haha, did you review yourself 💯